pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -27,10 +27,10 @@ var (
 	// proxy jobs do not have a trailing -version segment
 	proxyRegex   = regexp.MustCompile(`(?i)-proxy`)
 	promoteRegex = regexp.MustCompile(`(?i)^promote-`)
-	ppc64leRegex = regexp.MustCompile(`(?i)-ppc64le-`)
+	ppc64leRegex = regexp.MustCompile(`(?i)-ppc64le`)
 	rtRegex      = regexp.MustCompile(`(?i)-rt-`)
```

Comment:
Can we just remove the trailing `-` everywhere? Renaming to the new form will happen very soon for pretty much all jobs, just this morning we merged a PR that migrated the realtime jobs for instance (https://github.com/openshift/release/pull/16434)

Reply:
updated